packages/widgets/address-update/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "tocco-address-update",
+  "version": "0.1.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "compile:prod": "cd ../../../ && npm run compile:prod -- --package=address-update",
+    "prepublish": "npm run compile:prod"
+  },
+  "devDependencies": {
+    "tocco-app-extensions": "*",
+    "tocco-docs-browser": "*",
```

Comment:
I think without the dependency the local development is not working properly because the `tocco-docs-browser` is not in the node_modules folder from `address-update` and therefore the files cannot be found.
But yes, if don't need the `chooseDocument` at all, we can remove it completely from `address-update`. But I thought it's a nice "Standardfeature" for document types.